### ISSUE TITLE
Auo-fix on PRs

### ISSUE THREAD

**Issue by catenocrypt** [trustwallet/assets#3119]
It would be comfortable if fixes (e.g. add to whitelist, formatting) would happen automatically in a PR.

Running fixes and committing them should happen on the PR, on the source branch.

Related to #2694 .

**Comment by catenocrypt** [comment]
Difficulties:
- Commit requires a token, a secret token from main repo cannot be used in the forks
- Special action token GITHUB_TOKEN can be used, but
- to be able to checkin, one needs the original branch, while in the PR there is only the PR commit, no original branch
- The fix build can be on the original branch (on every branch; push not pull_request), and does the fix commit regardless of PR.  But
- fix checkin committed with Github.token does not trigger new CI builds, so check builds will not be executed on the last commit, and PR will not be able to merge (unless you are admin).  There is no way to trigger the build manually…

in short: fix build is executed, commits fixes (e.g. addition to whitelist), but PR will not be mergable.

**Comment by catenocrypt** [comment]
Fix-auto-commit disabled for now.

**Comment by catenocrypt** [comment]
Closed with the status:  Auto-fix commits happen on branches in all forked repos, with the limitations:
- auto-commit does not trigger CI run, so if this is the last commit on the PR, PR is not 'green'
- sometimes fix cannot be commited due to permission issuse
